code/kelsie/lab08/lab08_kelsie.py

```diff
@@ -0,0 +1,65 @@
+class ATM :
+    def __init__(self, balance=0 , interest=(1/1000)) :
+        self.balance = balance
+        self.interest = interest
+    
+    def check_balance(self) :
+        return self.balance
+
+    def deposit(self, amount) :
+        self.balance = self.balance =+ amount
```

Comment:
You have one symbol too many in your deposit function here on line 10. Test your program by running it, depositing some amount, then calculate interest. and then ask for the balance again. You will see a response that only returns the interest. 

Only one character needs to be removed, If you find yourself changing more than that then let me know.